projects/demo/src/modules/experimental/card/examples/18/index.less

```diff
@@ -0,0 +1,21 @@
+.image-dark {
+    background: url('/assets/images/road-illustration.jpg') no-repeat center;
+    background-size: cover;
+
+    [tuiTitle],
+    [tuiSubtitle],
+    .close {
```

Comment:
You can use `tuiTheme="night"` to automatically change vars instead

Reply:
fixed